src/components/datepicker/datepicker.html

```diff
@@ -1,48 +1,45 @@
 <div class="m-datepicker"
      :class="{ 'm--is-disabled': isDisabled,
-'m--is-readonly': isReadonly,
-'m--is-waiting': isWaiting,
-'m--has-error': calandarError,
-'m--is-valid': isValid,
-'m--has-validation-message': hasValidationMessage }"
+            'm--is-readonly': isReadonly,
+            'm--is-waiting': isWaiting,
+            'm--has-error': calandarError,
+            'm--is-valid': isValid,
+            'm--has-validation-message': hasValidationMessage }"
      :style="{ width: inputWidth, maxWidth: inputMaxWidth }">
     <m-input-style :label="label"
                    :label-for="id"
                    :disabled="isDisabled"
                    :readonly="isReadonly"
                    :waiting="isWaiting"
-                   :focus="open"
+                   :focus="isFocus"
```

Comment:
textuellement parlant un `hasFocus` est plus cohérent qu'un `isFocus`. Habituellement ces deux préfix sont interchangeables en fonction du type d'élément duquel on parle

Reply:
je suis d'accord mais pour l'instant isFocus provient de Input-style qui est out of scope de ce PR.